default/python/AI/AIstate.py

```diff
@@ -230,9 +230,10 @@ def get_prev_turn_uid(self):
         return self.turn_uids.get(fo.currentTurn() - 1, '0')
 
     def __refresh(self):
+        global invasionTargets
         """Turn start AIstate cleanup/refresh."""
         fleetsLostBySystem.clear()
-        invasionTargets[:] = []
+        invasionTargets = []
```

Comment:
No global declaration and ```invasionTargets.clear()``` strongly prefered.

Reply:
Fixed.